modules/ROOT/pages/sap/sap-connector-reference-44.adoc

```diff
@@ -0,0 +1,807 @@
+= SAP Connector Reference 5.0
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
+
+SAP Connector v4.4
+
+Release Notes: xref:release-notes::connector/sap-connector-release-notes-mule-4.adoc[SAP Connector Release Notes]
+
+The Anypoint Connector for SAP (SAP Connector) provides a class that has an overall configuration for the SAP extension.
```

Comment:
Kim says to not put "The" before the connector name.
Just: Anypoint Connector for SAP (SAP Connector)